source/reference/limitations.txt

```diff
@@ -24,35 +24,6 @@ Limitations
 General Limitations
 -------------------
 
-- The minimum supported server versions are MongoDB 6.0.8 and 7.0.0.
```

Comment:
I think it would be nice to have a bullet here saying something like "See the MongoDB Server Version Compatibility and Support section for more information." to indicate that server version compatibility stuff is located there